css/main.css

```diff
@@ -137,6 +137,12 @@ footer, #masthead {
 	text-shadow: 0 0 .5em #666;
 }
 
+.ui.card.author>.image>img {
+	max-height: 258px;
```

Comment:
Il ne faut [pas](http://stackoverflow.com/questions/609517/why-em-instead-of-px) spécifier de dimensions en pixels dans un layout élastique, sinon la mise en page sera brisée si le visiteur change la taille de sa police.
Surtout, cette propriété est en conflit avec les propriétés fournies par Semantic UI, et potentiellement en conflit avec `object-fit`.